### PR TITLE
Add indirect-funding actions

### PR DESCRIPTION
**Pull request by andrewgordstewart**
Adds some actions that will be routed to the `indirectFundingReducer`

### CODE REVIEW

File: packages/wallet/src/redux/indirect-funding/player-a/actions.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { WalletProcedure } from '../../types';
```
Comment (kerzhner) [inline packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: What are your thoughts on file and directory naming conventions? At the moment, we are snake casing file and directory names: `channelState`, `directFunding`, etc

File: packages/wallet/src/redux/indirect-funding/player-a/actions.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { Commitment } from 'fmg-core/lib/commitment';
+
+export const STRATEGY_APPROVED = 'WALLET.INDIRECT_FUNDING.STRATEGY_APPROVED';
+export const strategyApproved = (
+  channelId: string,
+  procedure: WalletProcedure.IndirectFunding,
+) => ({
+  type: STRATEGY_APPROVED as typeof STRATEGY_APPROVED,
+  channelId,
+  procedure,
```
Comment (kerzhner) [inline packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: Does this need to be a parameter? Is this action used for any other procedure?

Comment (andrewgordstewart) [reply packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: an action without a procedure can't be routed -- that's the point of the `procedure` property, isn't it?

File: packages/wallet/src/redux/indirect-funding/player-a/actions.ts (hunk 1 above)
Comment (andrewgordstewart) [reply packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: I think you mean camel casing. I guess I prefer camel casing, but was matching the directory names from this PR: https://github.com/magmo/apps/pull/285

I think I'll change them here, and request changes in #285

File: packages/wallet/src/redux/indirect-funding/player-a/actions.ts (hunk 2 above)
Comment (kerzhner) [reply packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: Never mind, I see what is happening here.

File: packages/wallet/src/redux/indirect-funding/player-a/actions.ts (hunk 1 above)
Comment (kerzhner) [reply packages/wallet/src/redux/indirect-funding/player-a/actions.ts]: 👍

**Pull request merged**